nei/classes/nei.py

```diff
@@ -1280,26 +1232,107 @@ def ionfrac_bar_plot(self, time_index):
 
             alpha = 1.0
             colors = ['blue', 'red']
+
+            #Color index counter
+            c_idx = 1
```

Comment:
Instead of `c_idx`, a possibility here would be to use a variable name like `color_index` or `col_idx` to make it easier to read later.  I tend to find longer variable names easier to read than abbreviations since I don't have to go back and look at where the variable was defined.